src/st/no1/e_wall_chicken_secret.c

```diff
@@ -22,7 +22,7 @@ void func_us_801BE880(Entity* self) {
         self->zPriority = 0x70;
         self->hitPoints = 0x7FFF;
         self->hitboxState = 0;
-        self->ext.et_801BE880.unk82 = 0;
+        self->ext.segmentedBreakableWall.damageTaken = 0;
```

Comment:
Updated this to share the ext from CAT. This whole file can be cleaned up further and potentially merged into shared header as the walls are very similar but I will look at doing that in a future PR